src/realm/object-store/sync/sync_session.cpp

```diff
@@ -482,8 +472,8 @@ void SyncSession::download_fresh_realm(sync::ProtocolErrorInfo::Action server_re
     }
 
     fresh_sync_session->assert_mutex_unlocked();
-    if (m_flx_subscription_store) {
-        sync::SubscriptionSet active = m_flx_subscription_store->get_active();
+    if (auto local_subs_store = get_flx_subscription_store(); local_subs_store) {
```

Comment:
nit: i think `if (auto local_subs_store = get_flx_subscription_store())` is enough